lib/Graph/PlaceholderBindings.cpp

```diff
@@ -64,26 +29,13 @@ Tensor *PlaceholderBindings::get(Placeholder *P) const {
   return it->second;
 }
 
-Placeholder *
-PlaceholderBindings::getPlaceholderByName(llvm::StringRef name) const {
```

Comment:
Let's keep this API. It can be useful.

Reply:
If its just for unittests it doesn't need to be fast though, we can still drop the `nameMap_` and just iterate over all placeholders. It would be nice to obviously discourage use of this API in runtime code.

Reply:
Sounds like a plan, I don't want to keep nameMap_ but iterating is fine